### PR TITLE
Week 5

### PR DESCRIPTION
**Pull request by abhijeetd700**
@anirao316

Assignment-1

Screenshot (187)

Assignment-2

Screenshot (188)

Assignment-3

Screenshot (189)

Assignment-4
Screenshot (190)

### CODE REVIEW

File: Week-5/assignment-4/package.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "name": "assignment-4",
+  "version": "1.0.0",
+  "description": "Assignment-4",
```
Comment (anilrao316) [inline Week-5/assignment-4/package.json:4]: remove package.json,gitignore files

File: Week-5/assignment-4/calculator.test.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+const cal = require('./calculator');
+
+describe('Calculator test',
+()=>{
+    test('Addition of 5 & 4 should be 9',()=>{
+        expect(cal.sum(5,4)).toBe(9);
```
Comment (anilrao316) [inline Week-5/assignment-4/calculator.test.js]: [For your learning] Add some negative test cases as well.. for ex : if you pass string to add method as in sum('1','2') result will be 12 right.. so add a functionality in the code such that data type is only numbers.. if the parameters are string return "number are not integers".. add this test case as well..

**Comment by anilrao316** [comment]
rest looks good!

File: Week-5/assignment-4/calculator.test.js (hunk 2 above)
Comment (abhijeetd700) [reply Week-5/assignment-4/calculator.test.js]: Done ...

Screenshot (192)

**Pull request merged**